src/modules/components/proposal/proposalDataListItem/approvalThresholdResult/approvalThresholdResult.tsx

```diff
@@ -0,0 +1,28 @@
+import { Progress, formatterUtils } from '../../../../../core';
+import { type IApprovalThresholdResult } from '../proposalDataListItemStructure';
+
+export interface IApprovalThresholdResultProps extends IApprovalThresholdResult {}
+
+/**
+ * `ApprovalThresholdResult` component
+ */
+export const ApprovalThresholdResult: React.FC<IApprovalThresholdResultProps> = (props) => {
+    const { approvalAmount, approvalThreshold } = props;
+    const percentage = Math.min((approvalAmount / approvalThreshold) * 100, 100);
```

Comment:
The `Progress` component already limits the value to be 0 < value < 100, I'd remove the `Math.min` here.
I would probably add a check for approvalThreshold to be !== 0

Reply:
I don't think it's possible to have a zero approvalThreshold. Also, if you've a proposal without people to vote on, this component crashing is the least of your issues lol